solvebio/resource/object.py

```diff
@@ -545,3 +549,19 @@ def untag(self, tags, dry_run=False, apply_save=True):
         """Remove tags on an object"""
 
         return self.tag(tags=tags, remove=True, dry_run=dry_run, apply_save=apply_save)
+
+    def query_object_content(self, limit=100, **_kwargs):
+
+        if not self.is_file:
+            raise SolveError('The functionality is only supported for files. '
+                             'This is a {}.'.format(self.object_type))
+
+        # TODO: limit param has been implemented in the API endpoint so far
+        params = {'limit': limit}
+
+        try:
```

Comment:
Is it necessary to try/except this if there is just a raise?

Reply:
sure, this happens when you copy/paste things without thinking about them.